requirements-dev.txt

```diff
@@ -0,0 +1 @@
+flake8
```

Comment:
It is nice to split the testing/development requirements from the install requirements.